2024-05-15 08:18:37,841 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Configure stats pid to 49986
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Loading settings from /Users/kithminrw/.config/wandb/settings
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Loading settings from /Users/kithminrw/Documents/GitHub/ml-nn-zero2hero/notebooks/01/wandb/settings
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_init.py:_log_setup():520] Logging user logs to /Users/kithminrw/Documents/GitHub/ml-nn-zero2hero/notebooks/01/wandb/run-20240515_081837-68o6i8ud/logs/debug.log
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/kithminrw/Documents/GitHub/ml-nn-zero2hero/notebooks/01/wandb/run-20240515_081837-68o6i8ud/logs/debug-internal.log
2024-05-15 08:18:37,845 INFO    MainThread:49986 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd830ba9f10>
2024-05-15 08:18:37,846 INFO    MainThread:49986 [wandb_init.py:init():560] calling init triggers
2024-05-15 08:18:37,846 INFO    MainThread:49986 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.02, 'architecture': 'CNN', 'dataset': 'CIFAR-100', 'epochs': 10}
2024-05-15 08:18:37,846 INFO    MainThread:49986 [wandb_init.py:init():610] starting backend
2024-05-15 08:18:37,846 INFO    MainThread:49986 [wandb_init.py:init():614] setting up manager
2024-05-15 08:18:37,851 INFO    MainThread:49986 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-15 08:18:37,857 INFO    MainThread:49986 [wandb_init.py:init():622] backend started and connected
2024-05-15 08:18:37,880 INFO    MainThread:49986 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-15 08:18:37,880 INFO    MainThread:49986 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-15 08:18:37,881 INFO    MainThread:49986 [wandb_init.py:init():711] updated telemetry
2024-05-15 08:18:37,932 INFO    MainThread:49986 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-15 08:18:38,195 INFO    MainThread:49986 [wandb_run.py:_on_init():2396] communicating current version
2024-05-15 08:18:38,338 INFO    MainThread:49986 [wandb_run.py:_on_init():2405] got version response 
2024-05-15 08:18:38,339 INFO    MainThread:49986 [wandb_init.py:init():795] starting run threads in backend
2024-05-15 08:18:45,083 INFO    MainThread:49986 [wandb_run.py:_console_start():2374] atexit reg
2024-05-15 08:18:45,084 INFO    MainThread:49986 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2024-05-15 08:18:45,084 INFO    MainThread:49986 [wandb_run.py:_redirect():2294] Wrapping output streams.
2024-05-15 08:18:45,084 INFO    MainThread:49986 [wandb_run.py:_redirect():2319] Redirects installed.
2024-05-15 08:18:45,087 INFO    MainThread:49986 [wandb_init.py:init():838] run started, returning control to user process
2024-05-15 08:18:45,089 INFO    MainThread:49986 [wandb_run.py:_finish():2103] finishing run kithminrw-ai/my-awesome-project/68o6i8ud
2024-05-15 08:18:45,090 INFO    MainThread:49986 [jupyter.py:save_history():445] not saving jupyter history
2024-05-15 08:18:45,090 INFO    MainThread:49986 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-15 08:18:45,090 INFO    MainThread:49986 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-15 08:18:45,090 INFO    MainThread:49986 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-15 08:18:45,090 INFO    MainThread:49986 [wandb_run.py:_restore():2326] restore
2024-05-15 08:18:45,091 INFO    MainThread:49986 [wandb_run.py:_restore():2332] restore done
2024-05-15 08:18:48,232 INFO    MainThread:49986 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-15 08:18:48,234 INFO    MainThread:49986 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-15 08:18:48,242 INFO    MainThread:49986 [wandb_run.py:_footer_sync_info():3953] logging synced files
